ehitchhiking-ui/src/app/shared/components/menu/menu.component.sass

```diff
@@ -42,6 +43,18 @@ footer
   time
     margin: 25%
 
+.toggle-button
+  margin-top: 2.2rem
+  padding: 0
+  min-width: 1.5rem
+  z-index: 3
+  position: fixed
+  .material-icons
+    height: 1.9rem!important
```

Comment:
Pls add constant for material-icons sizes. Is it possible override styles without !important?

Reply:
removed